src/app/zap-templates/zcl/data-model/chip/illuminance-measurement-cluster.xml

```diff
@@ -30,7 +30,7 @@ limitations under the License.
       <attribute side="server" code="0x0001" define="ILLUM_MIN_MEASURED_VALUE" type="INT16U" min="0x0001" max="0xFFFD" writable="false"                   isNullable="true"                  optional="false">MinMeasuredValue</attribute>
       <attribute side="server" code="0x0002" define="ILLUM_MAX_MEASURED_VALUE" type="INT16U" min="0x0002" max="0xFFFE" writable="false"                   isNullable="true"                  optional="false">MaxMeasuredValue</attribute>
       <attribute side="server" code="0x0003" define="ILLUM_TOLERANCE"          type="INT16U" min="0x0000" max="0x0800" writable="false"                                                      optional="true" >Tolerance</attribute>
-      <attribute side="server" code="0x0004" define="ILLUM_LIGHT_SENSOR_TYPE"  type="ENUM8"  min="0x00"   max="0xFF"   writable="false"                   isNullable="true" default="0xFF"   optional="true" >LightSensorType</attribute>
+      <attribute side="server" code="0x0004" define="ILLUM_LIGHT_SENSOR_TYPE"  type="LightSensorType"  min="0x00"   max="0xFF"   writable="false"                   isNullable="true" default="0xFF"   optional="true" >LightSensorType</attribute>
```

Comment:
Please do not make changes to XML that diverge it from the spec.  We've had way too much of this, and then the spec gets changed in a _different_ way and the XML has to get changed again, etc, and every change affects public APIs that people are using, which is bad.

In particular, the spec is likely to _not_ call the type of this thing LightSensorType.

The right ordering here is to get the spec to say the right things, then align the XML here to match.

Reply:
But do we really want the spec to explicitly define data type names is that not going too far? the fact the data type is called LightSensorType XML is not a spec violation as I see it, this is just a trait of the XML representation of the spec.

I would argue that the spec is more readable as it is, stating that this is an 8-bit enumeration, and this enumeration is defined in its own section in the spec.

But the fact ENUM8 type is translated into a uint8_t when ZAP is reading the XML files is a little dangerous.

Reply:
@bzbarsky-apple The specification changes are merged - https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/6469

Can this PR be approved now?